src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Machine> get() {
+        final Machine bulldozer1 = new Bulldozer("Ukraine",2023);
+        final Machine bulldozer2 = new Bulldozer("USA",2014);
+        final Machine bulldozer3 = new Bulldozer("Poland",2019);
+        bulldozer1.setColor("Yellow");
+        bulldozer1.setName("General purpose");
+        bulldozer2.setColor("Blu");
+        bulldozer2.setName("Specialized");
+        bulldozer3.setColor("Red");
+        bulldozer3.setName("Adapted");
```

Comment:
Please introduce all-args constructor to machines to unify code. As using both constructor and setters together looks strange.
Please recheck other classes
```suggestion
        final Machine bulldozer1 = new Bulldozer("Ukraine", 2023, "Yellow", "General purpose");
        final Machine bulldozer2 = new Bulldozer("USA", 2014, "Blu", "Specialized");
        final Machine bulldozer3 = new Bulldozer("Poland", 2019, "Red", "Adapted");
```